.github/workflows/push-asset-readme-update.yml

```diff
@@ -10,10 +10,10 @@ jobs:
     name: Push to trunk
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout code
```

Comment:
any reason this was removed?  they can be helpful when reviewing check runs that fail to see where things are going wrong

Reply:
You're correct, removed it mistakenly.